src/math_utils.rs

```diff
@@ -69,10 +78,10 @@ fn igcdex(num_a: BigInt, num_b: BigInt) -> (BigInt, BigInt, BigInt) {
     }
 }
 ///Finds a nonnegative integer x < p such that (m * x) % p == n.
-fn div_mod(n: BigInt, m: BigInt, p: BigInt) -> BigInt {
+pub fn div_mod(n: BigInt, m: BigInt, p: &BigInt) -> BigInt {
```

Comment:
`n` and `b` should probably be references as well.

Reply:
resolved:
- https://github.com/lambdaclass/cleopatra_cairo/pull/333/commits/db90895a67ed004b2280e763562e66168a408a7b
- https://github.com/lambdaclass/cleopatra_cairo/pull/333/commits/2e6aaad6adbd1f438678c587d5aa493fdebeb1c7